src/eherkenning/tests/test_mock_views.py

```diff
@@ -108,8 +108,14 @@ def test_get_returns_valid_response(self):
         self.assertContains(response, reverse("login"))
         self.assertNoEHerkenningURLS(response)
 
+    @patch(
+        "open_inwoner.kvk.signals.KvKClient.retrieve_rsin_with_kvk",
+        return_value="123456789",
+    )
```

Comment:
Let's make a habit of putting `auto_spec=True` in all the patches.

Reply:
good point, added them